src/containers/EventPage.tsx

```diff
@@ -40,12 +37,11 @@ export class EventPage extends React.Component<any, any> {
   }
 
   public render(): JSX.Element {
-    const {cmsData} = this.context;
+    const {cmsData} = this.props;
     return (
       <div className="pr-event-page">
           <span className="pr-event-page__content">
-            <CMSEditableForm isUserAuthorized={isUserAuthorized()}
-                          schema={this.getSchema()}
+            <CMSEditableForm schema={this.getSchema()}
                           formData={cmsData.event}
```

Comment:
I think we don't need to pass the formData here. The path is enough. By removing it we are reducing the chances of error (mistakenly passing formData from another path)